pkgs/top-level/all-packages.nix

```diff
@@ -15485,7 +15485,9 @@ in
 
   iputils = callPackage ../os-specific/linux/iputils { };
 
-  iptables = callPackage ../os-specific/linux/iptables { };
+  iptables = iptables-legacy;
+  iptables-legacy = callPackage ../os-specific/linux/iptables { };
+  iptables-compat = callPackage ../os-specific/linux/iptables { modeCompat = true; };
```

Comment:
Is it worth building this variant?

Reply:
@FRidh i set the legacy default mode of operation. Then can switch to a new compat mode.
Works normal on my home server.
Or leave only compat mode?

Reply:
On OS archlinux, debian 10 already is used iptables-compat

Reply:
I would just call this variant `iptables-nft` or `iptables-nftables` for clarity.

Reply:
In the OS Debian distribution, the package is also named iptables-compat
https://manpages.debian.org/testing/iptables-nftables-compat/iptables-compat.8.en.html

Reply:
Need to enable use iptables-compat to default?